pkg/controller/render/render_controller.go

```diff
@@ -75,15 +104,22 @@ func New(
 	mcpInformer mcfginformersv1.MachineConfigPoolInformer,
 	mcInformer mcfginformersv1.MachineConfigInformer,
 	ccInformer mcfginformersv1.ControllerConfigInformer,
+	isInformer imageinformersv1.ImageStreamInformer,
 	kubeClient clientset.Interface,
 	mcfgClient mcfgclientset.Interface,
+	imageClient imageclientset.Interface,
+	buildclient buildclientset.Interface,
+
```

Comment:
Lol I'm fairly confident this one is not just me being annoying :joy: @jkyros :heart: \n
```suggestion
```